src/main/frontend/handler/editor.cljs

```diff
@@ -1667,6 +1667,7 @@
    "{" "}"
    "(" ")"
    "`" "`"
+   "\"" "\""
```

Comment:
This is a non-markup symbol. Better impl it using a plugin.

Reply:
I think this is a useful feature that I found convenient in https://github.com/refined-github/refined-github
I am not sure if plugins can intercept input keys right now. Let me see if it can be implemented with a custom.js script ...